src/ansys/dpf/composites/layup_info/_layup_info.py

```diff
@@ -324,14 +324,28 @@ def get_analysis_ply_index_to_name_map(
     ----------
     mesh
         DPF Meshed region enriched with lay-up information
+
+    Note: Analysis plies of ACP's imported solid model which are linked
+    to homogeneous elements only are currently skipped.
```

Comment:
```suggestion
    .. note::
    Analysis plies of ACP's imported solid model that are linked only
    to homogeneous elements are currently skipped.
```
The ``note`` directive is supported in PY files. Using it will make this content stand out.